### PR TITLE
[Improvement-11380][scp-host.sh] Set StrictHostKeyChecking=no option to ssh

### LINKED ISSUE #11380: [Improvement][scp-host.sh] Set StrictHostKeyChecking=no option to ssh
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Even if the password-free account is configured, when the user uses the `scp-host.sh` script to log in to the machine for the first time, an error (Host key verification failed.) may occur.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rickchengx**
## Purpose of the pull request

This PR close: #11380 

Even if the password-free account is configured, when the user uses the scp-host.sh script to log in to the machine for the first time, an error (Host key verification failed.) may occur.

So set the ` -o StrictHostKeyChecking=no` option to ssh in `scp-hosts.sh`.

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Comment by SbloodyS** [comment]
Hi @rickchengx , to target specific issue from PR, we recommend use keyword fix: #issue_id or close: #issue_id or closes: #issue_id in your PR describe(not title, just desc). It would not only connect issue to PR but also close issue automatically when PR is be closed.

**Comment by rickchengx** [comment]
> Hi @rickchengx , to target specific issue from PR, we recommend use keyword fix: #issue_id or close: #issue_id or closes: #issue_id in your PR describe(not title, just desc). It would not only connect issue to PR but also close issue automatically when PR is be closed.

Sure. Thanks for your advice, I've added the keyword close: #issue_id.

File: script/scp-hosts.sh [hunk 1]
```diff
@@ -35,8 +35,8 @@ hostsArr=(${ips//,/ })
 for host in ${hostsArr[@]}
 do
 
-  if ! ssh -p $sshPort $host test -e $installPath; then
-    ssh -p $sshPort $host "sudo mkdir -p $installPath; sudo chown -R $deployUser:$deployUser $installPath"
+  if ! ssh -o StrictHostKeyChecking=no -p $sshPort $host test -e $installPath; then
+    ssh -o StrictHostKeyChecking=no -p $sshPort $host "sudo mkdir -p $installPath; sudo chown -R $deployUser:$deployUser $installPath"
```
Comment (SbloodyS) [inline script/scp-hosts.sh:39]: Would you please add this to the `script/status-all.sh`?

Comment (rickchengx) [reply script/scp-hosts.sh:39]: > Would you please add this to the `script/status-all.sh`?

Sure.

**Review by SbloodyS — APPROVED** [review]
LGTM. Thanks.

**Review by EricGao888 — APPROVED** [review]

**Pull request merged**